### PR TITLE
[MOD-66] Changed layout of Save/Delete buttons, added Cancel Button

### PR DESCRIPTION
**Pull request by enymuss**
https://issues.openmrs.org/browse/MOD-66

### CODE REVIEW

File: app/js/showControllers.js [hunk 1]
```diff
@@ -46,6 +46,10 @@ angular.module('modulusOne.showControllers', [
       })
     }
 
+    $scope.cancelEdit = function(toggleEditWhenDone) {
+      window.location.reload();
```
Comment (enymuss) [inline app/js/showControllers.js]: This probably isn't the proper way to do it.

Comment (elliottwilliams) [inline app/js/showControllers.js]: Yeah, try the `$state.reload()` method of the view router we use:
https://angular-ui.github.io/ui-router/site/#/api/ui.router.state.$state

**Comment by elliottwilliams** [comment]
This looks pretty good! I've got a couple UI suggestions, though:

* Only show the "Delete All Module Data" button when the module is in "edit mode". Since the delete button is big and red, I think it draws too much attention to itself when it's visible in the module view.

* Make the row of edit buttons (edit, cancel, delete, etc.) the same width as the table below. You can do this by using an 8 column grid: `<div class="col-md-8"> ... </div>`. This way, the `pull-right` on the delete button won't move it to the far right of the page. The UI is very left-aligned so I think it looks a little weird all the way to the right (on my screen at least)

Thanks for the good work! Let me know what you think about these.

File: app/js/showControllers.js [hunk 2]
```diff
@@ -46,6 +46,13 @@ angular.module('modulusOne.showControllers', [
       })
     }
 
+    $scope.cancelEdit = function(toggleEditWhenDone) {
+      $state.transitionTo($state.current, $stateParams, {
+        reload: true, inherit: false, notify: true
+      });
+
```
Comment (enymuss) [inline app/js/showControllers.js]: $state.reload worked well unless you didn't "tick" the changes. $state.reload is an alias for $state.transitionTo (https://angular-ui.github.io/ui-router/site/#/api/ui.router.state.$state#methods_reload). $state.transitionTo function has the desired effect.

File: app/partials/showModule.html [hunk 3]
```diff
@@ -4,13 +4,22 @@
...
+
+    <button class="btn btn-link pull-left" ng-click="cancelEdit(true)">
+      Cancel
+    </button>
+
+    <button class="btn btn-danger pull-right" ng-click="confirmDeleteModule()">
+      <span class="glyphicon glyphicon-trash"></span>Delete All Module Data
+    </button>
+
+  </div>
```
Comment (enymuss) [inline app/partials/showModule.html]: Put it into a div as suggested. Also put the "ng-switch-when" attributes inside since it was repeating.

**Comment by elliottwilliams** [comment]
Awesome, thanks for those changes, I'll merge this. There's currently a slight glitch when you press "cancel" due to the autoscroll caused by [MOD-87][0] (which I just reported). However, either you or I can get this bug fixed and it'll be okay by the time we do another release of Modulus UI.

[0]: https://issues.openmrs.org/browse/MOD-87

**Pull request merged**